turbo.json

```diff
@@ -13,45 +21,18 @@
     },
     "test": {
       "cache": false,
-      "persistent": true
-    },
-    "antlr4-c3#build": {
-      "dependsOn": ["antlr4#build"]
+      "persistent": true,
+      "dependsOn": ["build"]
     },
-    "@neo4j-cypher/language-support#build": {
-      "dependsOn": ["antlr4#build", "antlr4-c3#build"]
-    },
-    "@neo4j-cypher/schema-poller#build": {
-      "dependsOn": ["@neo4j-cypher/language-support#build"]
-    },
-    "@neo4j-cypher/language-server#build": {
-      "dependsOn": [
-        "@neo4j-cypher/language-support#build",
-        "@neo4j-cypher/schema-poller#build"
-      ]
-    },
-    "@neo4j-cypher/react-codemirror#build": {
-      "dependsOn": ["@neo4j-cypher/language-support#build"]
-    },
-    "@neo4j-cypher/react-codemirror#test:e2e": {
+    "test:e2e": {
       "cache": false,
       "persistent": false,
-      "dependsOn": ["@neo4j-cypher/react-codemirror#build"]
-    },
-    "@neo4j-cypher/react-codemirror-playground#build": {
-      "dependsOn": ["@neo4j-cypher/react-codemirror#build"]
+      "dependsOn": ["build"]
     },
     "@neo4j-cypher/react-codemirror-playground#dev": {
-      "dependsOn": ["@neo4j-cypher/react-codemirror#build"]
-    },
-    "neo4j-for-vscode#build": {
-      "persistent": false,
-      "dependsOn": ["@neo4j-cypher/language-server#build"]
-    },
-    "neo4j-for-vscode#test:e2e": {
-      "cache": false,
-      "persistent": false,
-      "dependsOn": ["@neo4j-cypher/react-codemirror#test:e2e"]
+      "outputs": [],
+      "dependsOn": ["^build"],
```

Comment:
Is this correct? From the [docs](https://turbo.build/repo/docs/core-concepts/monorepos/running-tasks#turborepo-can-multitask) of Turbo, they say this makes build act on the dependencies before running build? Why do we need that?

```
      // ^build means `build` must be run in dependencies
      // before it can be run in this workspace
```

Reply:
When I open the codemirror debug and I change something in language-support, I'd expect the build to reload, but it doesn't? 🙈

Reply:
This means that the `build` scripts in the dependencies must run before the `dev` script can run, since we'd need `@neo4j-cypher/react-codemirror` to be built. As I read the docs, the `^scriptName` syntax is "run scriptName" in in dependencies first and the example happens to be build + build

Second: Turbo doesn't support watching for file changes.. I mentioned this in the PR description as one of the main reasons to switch to `wireit` or some other simpler&better tool